lib/index.js

```diff
@@ -8,6 +8,7 @@ var Spawn = require("child_process").spawn
   , Ul = require("ul")
   , OneByOne = require("one-by-one")
   , Sliced = require("sliced")
+  , Q = require('q')
```

Comment:
I am trying to make my transpiler modular so things like parsing, building cobol AST, rewriting, translating to C# are totally separated so I am sure that by it should be rather easy to translate to JS by creating new target.